pkg/datapath/linux/node_linux_test.go

```diff
@@ -1026,7 +1025,6 @@ func (s *linuxPrivilegedIPv4OnlyTestSuite) TestArpPingHandling(c *check.C) {
 	}
 	// wait 1 second to give the OS time to setup the routes for the veth pairs
 	// just created.
```

Comment:
at least I know I'm not the only one that doesn't read comments... You forgot to remove these 2 lines as well.